prog/download_boot_image.rb

```diff
@@ -30,9 +38,10 @@ def custom_url
       sshable.cmd("common/bin/daemonizer --clean download_#{image_name.shellescape}")
       hop_learn_storage
     when "NotStarted"
-      sshable.cmd("common/bin/daemonizer 'host/bin/download-boot-image #{image_name.shellescape} #{custom_url.shellescape}' download_#{image_name.shellescape}")
+      url = custom_url || blob_storage_client.get_presigned_url("GET", Config.ubicloud_images_bucket_name, "#{image_name}-#{vm_host.arch}.raw").to_s
```

Comment:
bit surprise there's no expiry date here; defaulted?

Reply:
It was set to default. I removed the default value and assigned it explicitly.